src/connector.rs

```diff
@@ -55,7 +55,7 @@ impl TlsConnector {
     /// assert!(data.ends_with("</html>") || data.ends_with("</HTML>"));
     /// #
     /// # Ok(()) }) }
-    pub async fn connect<S>(&self, domain: &str, stream: S) -> Result<TlsStream<S>, Error>
+    pub(crate) async fn connect<S>(&self, domain: &str, stream: S) -> Result<TlsStream<S>, Error>
```

Comment:
the example above should go away in this case I think

Reply:
done!